src/services/UriBuilder.ts

```diff
@@ -0,0 +1,37 @@
+import { ParameterPlace } from "../models/kinds/ParameterPlace";
+import { IMethodModel, IMethodParameterModel } from "../models/MethodModel";
+import { lowerFirst } from "../utils";
+
+export class UriBuilder {
+    public buildUri(model: IMethodModel): string {
+        const queryParams = this.getQueryParams(model.parameters);
+        const pathParams = this.getPathParam(model.parameters);
+        let path = lowerFirst(model.originUri);
```

Comment:
i guess you don't need change origin uri

Reply:
Done